rush.json

```diff
@@ -581,6 +581,16 @@
       "packageName": "@azure/digital-twins",
       "projectFolder": "sdk/digitaltwins/digital-twins",
       "versionPolicyName": "client"
+    },
+    {
+      "packageName": "@azure/batch",
+      "projectFolder": "sdk/batch/batch",
+      "versionPolicyName": "client"
+    },
+    {
+      "packageName": "@azure/arm-batch",
+      "projectFolder": "sdk/batch/arm-batch",
+      "versionPolicyName": "client"
```

Comment:
Hey @xingwu1 

We have two sets of packages in this repo using two separate build and release pipelines.
- A set of completely hand written packages or auto generated packages with a hand written layer over it.
- Auto generated packages

The first kind uses rush and will need an entry in this file, the second kind does not use rush and you can use npm to build the package and run its tests. Both `@azure/batch` and `@azure/arm-batch` packages fall under the second category. Can you please revert the changes to rush.json and the pnpm-lock.yaml files in this PR?

@deyaaeldeen Am guessing @xingwu1 landed in this situation by following our contributing guide which talks about using only rush and does not talk about contributing to any of the auto generated packages. We should have a section on the second kind so that they don't get added to rush in this manner

Reply:
Okay, I will fix it.

Reply:
@ramya-rao-a done.

Reply:
Yes. I was following the contibuting guide. I will revert these changes.